src/styles/variables.ts

```diff
@@ -142,7 +142,7 @@ export default {
     signInLogoWidthLargeScreen: 144,
     signInLogoHeightLargeScreen: 108,
     signInLogoWidthPill: 132,
-    tabSelectorButtonHeight: 40,
+    tabSelectorButtonHeight: 42,
```

Comment:
@akinwale Just wanted to mention why this change. At highest setting the line height becomes 21px and there is a vertical padding of 24px for tabSelectorButton. So at 40px the text still clips a bit. That's the reason for increasing this to 42px.